backend/src/main/java/com/htecgroup/skynest/service/impl/UserServiceImpl.java

```diff
@@ -145,4 +146,17 @@ public void changePassword(UserChangePasswordRequest userChangePasswordRequest,
     Email email = EmailUtil.createPasswordChangeNotificationEmail(changedPasswordDto);
     emailService.send(email);
   }
+
+  @Override
+  public void promoteUser(UUID userId) {
+    UserDto userDto = findUserById(userId);
```

Comment:
I would add that in a custom annotation, maybe that PR is already merged, so you can see how to do it